WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -186,8 +186,14 @@ - (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDiction
 
     if ([url isKindOfClass:[NSURL class]] && [[url absoluteString] hasPrefix:WPComScheme]) {
         NSString *URLString = [url absoluteString];
-            
-        if ([URLString rangeOfString:@"magic-login"].length) {
+        if ([URLString rangeOfString:@"new_user"].length) {
```

Comment:
Yea, that's directly from whatever creates the URL. FWIW, I'm removing that check anyway in the Epilogue since we have a flag that indicates if the email was for login for signup.